src/xmipp/applications/scripts/cone_deepalign_predict/batch_cone_deepalign_predict.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import os
+import sys
+import xmippLib
+
+
+def loadData(mdIn, mdExp):
+    XProj=None
+    XExp=None
+    Nproj = mdIn.size()
+    Nexp = mdExp.size()
+    idx = 0
+    for objId in mdIn:
+        fnImg = mdIn.getValue(xmippLib.MDL_IMAGE,objId)
+        I = xmippLib.Image(fnImg)
+        if XProj is None:
+            Xdim, Ydim, _, _ = I.getDimensions()
+            XProj = np.zeros((Nproj,Xdim,Ydim,1),dtype=np.float64)
+        XProj[idx,:,:,0] = I.getData()
+        idx+=1
+
+    idx = 0
+    for objId in mdExp:
```

Comment:
duplicated code (line 15)

Reply:
They are sufficiently different. I agree that this part could be sent to a function, but at the moment, Amaya is no longer here to fix it.